extensions/wikia/Track/Track.php

```diff
@@ -112,7 +111,7 @@ public static function event ($event_type, $param=null) {
 	protected static function shouldTrackEvents() {
 		// Check request headers to make sure this is not a mirrored
 		// request, which could result in duplicate tracking events
-		return F::app()->wg->request->getHeader( self::HEADER_MIRRORED_REQUEST ) === false;
+		return F::app()->wg->request->isWikiaInternalRequest() === false;
```

Comment:
Pinging @garthwebb - let's use a more generic header here (which will cover all internal requests - sent by DC warmer and MediaWiki itself via `Http::get`)